test/e2e/reboots/kernel_parameter_add_rm.go

```diff
@@ -33,13 +33,15 @@ var _ = ginkgo.Describe("[reboots][kernel_parameter_add_rm] Node Tuning Operator
 		ginkgo.AfterEach(func() {
 			// This cleanup code ignores issues outlined in rhbz#1816239;
 			// this can cause a degraded MachineConfigPool
+
+			// Ignore failures to cleanup resources which are already deleted or not yet created.
 			ginkgo.By("cluster changes rollback")
 			if node != nil {
-				util.ExecAndLogCommand("oc", "label", "node", "--overwrite", node.Name, nodeLabelRealtime+"-")
+				_, _, _ = util.ExecAndLogCommand("oc", "label", "node", "--overwrite", node.Name, nodeLabelRealtime+"-")
```

Comment:
the linter cannot ignore these at all ?     I hope this will not complicated automated cherry picks too much

Reply:
You can exclude these checks by something like:
```
    - linters:
        - errcheck
      text: "Error return value of .* is not checked"
```
however, I'm not sure I'm in favour of during that as it will mask a lot of cases where the checking is warranted.  This PR includes so many fixes and changes that the backports will be difficult anyway.

Reply:
Personally, in the future, I believe it is worth adding even more linters.  Look [at the benefit so far](https://github.com/openshift/cluster-node-tuning-operator/pull/1253/commits/0469f598bbf10e588b3fd7dd4e3b6ccee206d5d0) -- we might catch more things.

Reply:
ok . lets align with the linter